catalog-entities/marketplace/plugins/all.yaml

```
@@ -54,6 +54,7 @@ spec:
     - ./notifications.yaml
     - ./notifications-email.yaml
     - ./openshift-cluster-manager.yaml
+    - ./optimizations.yaml
```

```suggestion
    - ./redhat-resource-optimization.yaml
```
This change also requires renaming the associated file.